pkg/image/registry/imagesignature/registry.go

```diff
@@ -0,0 +1,54 @@
+package imagesignature
+
+import (
+	kapi "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/rest"
+	"k8s.io/kubernetes/pkg/api/unversioned"
+	"k8s.io/kubernetes/pkg/runtime"
+
+	"github.com/openshift/origin/pkg/image/api"
+)
+
+// Registry is an interface for things that know how to store ImageStreamImage objects.
+type Registry interface {
```

Comment:
We've been skipping these lately.  All access goes through a client from `pkg/client` which hits `RESTStorage` serverside.